__tests__/apiSpec.tests.js

```diff
@@ -0,0 +1,19 @@
+const request = require('supertest');
+
+const app = require('../Server/index.js');
+
+describe("Test the root path", () => {
```

Comment:
I think this is not a valid test, since that respond is already build-in in the framework

Reply:
@jgenso1690 basically the test is checking to make sure the static files are being sent, so I'm clear, are you saying since I'm using app.use(express.static) its not necessary to test?